Core/include/Acts/Utilities/detail/Line3DWithPartialDerivatives.ipp

```diff
@@ -10,11 +10,24 @@
 
 #include "Acts/Utilities/detail/Line3DWithPartialDerivatives.hpp"
 
-#include "Acts/Definitions/Common.hpp"
 #include "Acts/Utilities/UnitVectors.hpp"
+#include "Acts/Utilities/VectorHelpers.hpp"
 
 namespace Acts::detail {
 
+template <std::floating_point T>
+Line3DWithPartialDerivatives<T>::ParamVector
+Line3DWithPartialDerivatives<T>::parameters() const {
+  ParamVector pars{};
+  pars[static_cast<std::uint8_t>(ParIndex::x0)] = m_pos[Acts::eX];
```

Comment:
there is a helper if you prefer https://github.com/acts-project/acts/blob/2824422621b2ea6358c6f20592abd62bdf289920/Core/include/Acts/Utilities/Helpers.hpp#L186-L189